src/pages/Home/Home.tsx

```diff
@@ -1,13 +1,36 @@
-import { useCallback, useContext, useMemo, useState } from 'react';
+import {
+  // Fragment,
+  useCallback,
+  useContext,
+  // useEffect,
+  useMemo,
+  useState,
+} from 'react';
+import {
+  RotateCw,
+  // CircleAlert,
+  // Search,
+  // Loader,
+  // ListFilter,
+  LogOutIcon,
+  PlusIcon,
+  // Heart,
```

Comment:
já não está sendo usado podemos deixar mais limpo
```suggestion
import {
  useCallback,
  useContext,
  useMemo,
  useState,
} from 'react';
import {
  RotateCw,
  LogOutIcon,
  PlusIcon
```